tests/Regression/GitHub/2725-separate-class-before-after-pid.phpt

```diff
@@ -12,7 +12,6 @@ PHPUnit\TextUI\Command::main();
 PHPUnit %s by Sebastian Bergmann and contributors.
 
 ..                                                                  2 / 2 (100%)
-@afterClass output - PID difference should be zero: 0
```

Comment:
Please explain why (the expectation of) an existing test case had to be modified.